iati/core/data.py

```diff
@@ -154,6 +154,36 @@ def _raw_source_at_line(self, line_number):
         except IndexError:
             raise ValueError
 
+    @property
+    def version(self):
+        """Finds the version that this Dataset is specified against.
+
+        Returns:
+            str or None: The version of this Dataset. None if the version cannot be detected.
+
+        Todo:
+            Consider if this should raise an error if the Dataset is specified at a version that does not exist.
+
+        """
+        root_tree = self.xml_tree.getroot()
+        assumed_version_if_no_version_stated = '1.01'
+        version_iati_activities = root_tree.get('version', assumed_version_if_no_version_stated)
```

Comment:
Totally forgot about org files - but yep - it's exactly the same principle, so it will work in both cases. Will rename the variables to `version_iati_root` and `version_iati_child`.